skiko/build.gradle.kts

```diff
@@ -291,6 +289,11 @@ kotlin {
 
         val awtMain by getting {
             dependsOn(jvmMain)
+            languageSettings.optIn("org.jetbrains.skiko.InternalSkikoApi")
+
+            dependencies {
+                implementation("net.bytebuddy:byte-buddy-agent:1.12.22")
```

Comment:
Ah, when I checked ByteBuddy I missed this bit. That's unfortunate.

IDEA builds do have the open directives we need, so we don't need BB there. On the other hand, this would mean we need to document this need for all other scenarios. This meand that any CfD application that runs on the JBR will not work unless they add the opens directive (you can't easily figure out whether the opens directive is present). I can look into adding an agent to inject the directives without using ByteBuddy: it should be possible, as BB doesn't use any provate APIs or particular black magic. It'll just take more code.

Reply:
I totally forgot this needed doing, sorry 😔

I need to find some time to create the agent without ByteBuddy and see if it works, but a starting point would be to just attempt accessing the APIs and erroring out if they aren't accessible. Do you think it's reasonable asking folks to add an opens directive to the command line to run their projects? I can have a very explicit explanation in the error message.

Reply:
Removed the ByteBuddy dependency and code

Reply:
It was removed in a52b316549b770d7998e7af940919750d725d542